### ISSUE TITLE
[Bug]: [BAD_HTTP_STATUS: 403]  IP block or streaming URL deciphering ON ALL VIDEOS

### ISSUE THREAD

**Issue by sqrxz** [FreeTubeApp/FreeTube#7070]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [x] I have [searched the issue tracker for open and closed issues](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+sort%3Arelevance-desc) that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.

### Describe the bug

whenever I try to view any video I get this message.
[BAD_HTTP_STATUS: 403] Potential causes: IP block or streaming URL deciphering failed
I am not using a VPN, Invidious proxy and am on Local API
it was working ok yesterday

### Expected Behavior

watching videos

### Issue Labels

accessibility issue

### FreeTube Version

Release 0.23.2 Beta 

### Operating System Version

Win 10 22H2

### Installation Method

.zip / .7z

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

Release 0.23.2 Beta 

### Additional Information

Thank you for your time reading this, I greatly admire and appreciate all the work on the app, keeping up with the demand for so many years as a small team and adding new features while fighting google's blocking now that the team has grown

### Nightly Build

- [x] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
duplicate of https://github.com/FreeTubeApp/FreeTube/issues/6912
